Source/WebKit/Shared/API/glib/WebKitUserMessage.cpp

```diff
@@ -291,6 +291,8 @@ GUnixFDList* webkit_user_message_get_fd_list(WebKitUserMessage* message)
  * @message: a #WebKitUserMessage
  * @reply: a #WebKitUserMessage to send as reply
  *
+ * Send a reply to @message.
```

Comment:
Some of those would be nice to write without `@var` and `#type` markup. For example in this particular case I think something like this would be neater:

```suggestion
 * Send a reply to an user message.
```

Anyway, we can keep the focus of this patch in having brief descriptions, and leave actual content edits for a separate patch. In many cases one would need to know an understanding of how the API functions work, which is something that we can't expect from new contributors 😉&mdash;so I am mostly writing this down to help ourselves remember later about this.